releasenotes/notes/25818.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: release-notes/v2
+
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# kind describes the type of change that this represents. 
+# Valid Values are:
+# - bug-fix -- Used to specify that this change represents a bug fix. 
+# - security-fix -- Used to specify that this change represents a security fix. 
+# - feature -- Used to specify a new feature that has been added. 
+# - test -- Used to describe additional testing added. This file is optional for
+#   tests, but included for completeness.
+kind: security-fix
+
+# area describes the area that this change affects. 
+# Valid values are:
+# - traffic-management
+# - security
+# - telemetry
+# - installation
+# - istioctl
+# - documentation
+area: security
+
+# issue is a list of GitHub issues resolved in this note.
+issue: 
```

Comment:
There's no issue associated with this right?

Reply:
Nope. Should I create one?